core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/Application.java

```diff
@@ -1,13 +1,19 @@
 package io.metadew.iesi.server.rest;
 
 
+import lombok.extern.log4j.Log4j2;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.scheduling.annotation.EnableAsync;
+
 
 @SpringBootApplication
+@EnableAsync
+@Log4j2
 public class Application {
 
-	public static void main(String[] args) throws Exception {
+	public static void main(String[] args) {
 		SpringApplication.run(Application.class, args);
 	}
+
```

Comment:
Reformat file : remove extra blank lines.

Reply:
it's just an extra line to make the code more clear :)